packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -360,12 +360,77 @@ module.exports = ctx => ({
 
     // add redir url for main guide pages
     let found = guidesInfo.guideInfo[path];
+
+    if (path.endsWith('/main/')) {
+      // For paths such as /docs/guides/{guide-name}/main/ where the guide has stack selector/frameworks
+      let mainPagePath = path.slice(0, path.length - 5);
```

Comment:
Magic numbers are bad in general and it's also hard to find a compact alternative. Would something like this work to at least highlight the part we're removing?
```suggestion
      let mainPagePath = path.slice(0, -'main/'.length);
```
the same for line 394